### PR TITLE
Check English and clarify docstrings

### PR DESCRIPTION
**Pull request by merelvdthiel**
Closes #

## Proposed Changes

  -
  -
  -

## Change Type

- [ ] `bugfix` (+0.0.1)
- [ ] `minor` (+0.1.0)
- [ ] `major`  (+1.0.0)
- [ ] `refactoring` (no version update)
- [ ] `test` (no version update)
- [ ] `infrastructure` (no version update)
- [ ] `documentation` (no version update)
- [ ] `other`

## Checklist before review

- [ ] I added everything I wanted to add to this PR.
- [ ] \[Code or tests only\] I wrote/updated the necessary docstrings.
- [ ] \[Code or tests only\] I ran and passed tests locally.
- [ ] \[Documentation only\] I built the docs locally.
- [ ] My contribution is harmonious with the rest of the code: I'm not introducing repetitions.
- [ ] My code respects the adopted style, especially linting conventions.
- [ ] The title of this PR is explanatory on its own, enough to be understood as part of a changelog.
- [ ] I added or indicated the right labels.

- [ ] I added information regarding the timeline of completion for this PR.

- [ ] Please, comment on my PR while it's a draft and give me feedback on the development!

### CODE REVIEW

**Review by RazkinMalen — APPROVED** [review]

File: phys2cvr/regressors.py [hunk 1]
```diff
@@ -29,19 +29,19 @@
 
 def create_legendre(degree, length):
     """
-    Producesthe Legendre polynomials of order `degree`.
+    Produces Legendre polynomials of order `degree`.
```
Comment (smoia) [inline phys2cvr/regressors.py:32]: One thing that I didn't tell you is that the first line of the numpydoc has to be an imperative statement with a final dot.

```suggestion
    Produce Legendre polynomials of order `degree`.
```

File: phys2cvr/regressors.py [hunk 2]
```diff
@@ -146,7 +146,7 @@ def fine_shift(
     legacy=False,
 ):
     """
-    Compute fine shifts.
+    Compute fine shifts to further optimize shifts.
```
Comment (smoia) [inline phys2cvr/regressors.py:149]: Maybe we can write "[...] shifts locally"?

**Review by smoia — CHANGES_REQUESTED** [review]
It's great, thanks! There's just a little change to do

File: phys2cvr/regressors.py [hunk 3]
```diff
@@ -29,19 +29,19 @@
...
 def create_legendre(degree, length):
     """
-    Producesthe Legendre polynomials of order `degree`.
+    Produces Legendre polynomials of order `degree`.
 
     Parameters
     ----------
     degree : int
-        Highest order desired.
+        Highest number of desired orders.
```
Comment (smoia) [inline phys2cvr/regressors.py:37]:
```suggestion
        Highestest number of desired orders.
```

**Review by smoia — APPROVED** [review]
LGTM!

**Pull request merged**